rationale.md

```diff
@@ -15,19 +15,20 @@ As discussed in [issue 3](https://github.com/w3c/wot-scripting-api/issues/3), an
 Resolutions:
 - The browser implementations of the WoT Scripting API uses a namespace object `wot` in the browser.
 - Non-browser implementations that use various runtimes may use either a namespace object `wot`, or an API object provided by the `require()` or `import()` or similar mechanisms.
-- The `ConsumedThing` and `ExposedThing` objects are created by factory methods.
-
+- The `ConsumedThing` and `ExposedThing` objects are expected to be created by factory methods, though constructors are defined.
+- Interaction data can be retrieved with an attempted conversion by implementation as convenience interface for most JavaScript types, or as streams the applications can interpret.
+- Errors during protocol operations are exposed to applications.
 
 ## Discovery API
 
-Based on [WoT Current Practices](https://w3c.github.io/wot/current-practices/wot-practices.html#td-discovery), there are different discovery types: local (to the hardware), proximity based (such as BLE or NFC), registry (directory) based, and broadcast/multicast based. The discovery type is specific to the underlying protocol bindings.
+Represents the second stage of discovery in the [2-stage discovery process](https://github.com/w3c/wot-discovery/blob/master/proposals/directory.md).
```

Comment:
I would add what the second that is about like

Represents the second stage of discovery (service that provides querying capabilities) in the ...

Reply:
Amended the original commit with the extra addition.